src/pages/Home.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import BookList from '../components/BookList';
+
+const Home = () => (
+  <div>
+    <BookList />
```

Comment:
- [ ] You should have a form to create new books in your page, I noticed you did not add it. Kindly fix that 👍